python/humbug/report.py

```diff
@@ -87,6 +87,20 @@ def __init__(
         self.is_excepthook_set = False
         self.is_loggerhook_set = False
 
+    def _extract_error_base(self, error: Exception) -> List[str]:
```

Comment:
This is not the core functionality we require - this can be an optional parameter in `error_report` - in case a user wants to add a tag with exception base classes.

You should modify this function to return exception base classes up to a fixed recursion depth - some projects can have complicated exception class hierarchies.

Might be good to use `error_type: type` instead of `error: Exception`. You can do everything you need with the class.

Reply:
We might use this in the future. I suggest tagging this commit so that we can cherry pick into future work.